content/control-api/index.textile

```diff
@@ -0,0 +1,229 @@
+
+From the "My access tokens" area you can click the "Copy Token" button, to copy the button to the clipboard.
+
+You use the access token to authenticate requests to the Control API. To do this, you supply the access token as a Bearer token in the Authorization header of the HTTP request. For example, in the folllowing Curl request replace `<YOUR_ACCESS_TOKEN>` with the token you have copied to the clipboard:
+
+```[sh]
+curl -H 'Authorization: Bearer <YOUR_ACCESS_TOKEN>' ...
+```
+
+h3(#obtain-token). Obtain token details using the Control API
+
+You can use the Control API to obtain your token details. This is shown in the following request:
+
+```[sh]
+curl --location --request GET 'https://control.ably.net/v1/me' \
+--header 'Authorization: Bearer <YOUR_ACCESS_TOKEN>'
+```
+
+Sample response:
+
+```[json]
+{
+    "token": {
+        "id": "a975eecd-b189-4f5b-9f07-1197f3407193",
+        "name": "Control API - new token",
+        "capabilities": [
+            "write:namespace",
+            "read:namespace",
+            "write:queue",
+            "read:queue",
+            "write:rule",
+            "read:rule",
+            "write:key",
+            "read:key",
+            "write:app",
+            "read:app"
+        ]
+    },
+    "user": {
+        "id": 12140,
+        "email": "example@email.com"
+    },
+    "account": {
+        "id": "VgQpOZ",
+        "name": "Free account"
+    }
+}
+```
+
+Note that as well as the token details, the user and account of the token owner are also returned.
+
+h2(#ids). Account and application IDs
+
+Operations that affect your entire account, such as "listing the applications":/control-api/api-reference/#operation/listApps associated with that account, require an Account ID. Those that affect individual applications, such as "creating an API key":/control-api/api-reference/#tag/keys/paths/~1apps~1{app_id}~1keys/post , require an Application ID.
+
+h3(#account-id). How to find your Account ID
+
+In the "Ably dashboard":https://ably.com/accounts/any, on the top menu bar, select your account from the dropdown list and then select "Account settings": 
+
+<a href="/images/screenshots/control-api/account-settings-menu-item.png" target="_blank">
+  <img src="/images/screenshots/control-api/account-settings-menu-item.png" style="width: 40%" alt="Account Settings">
+</a>
+
+Your Account Settings are displayed.
+
+h3(#app-id). How to find your Application ID
+
+In the "Ably dashboard":https://ably.com/accounts/any select the app you want to find the application ID for. Click on the "Settings" tab:
+
+<a href="/images/screenshots/control-api/application-settings.png" target="_blank">
+  <img src="/images/screenshots/control-api/application-settings.png" style="width: 70%" alt="Application Settings">
+</a>
+
+The App ID is displayed under "Application settings". It is also the first part of your API key for that application. For example, if your API key is @28AB6c.DEFi0Q@, then the App ID is @28AB6c@.
```

Comment:
Perhaps link [here](https://knowledge.ably.com/what-is-an-app-api-key)?

Reply:
Yes.